lib/tasks/dad_tasks.rake

```diff
@@ -1,437 +1,447 @@
-# frozen_string_literal: true
```

Comment:
diff isn't helpful here... could you summarize what was changed?

Reply:
```
    # STEP 6: Create new Houses if it is the start of the month
    if Time.now.utc.to_date == Time.now.utc.at_beginning_of_month.to_date
      (1..3).each do |i|
        House.create(
          house_name: "House #{i}",
          house_start: Time.now.utc.at_beginning_of_month.to_date,
        )
      end
    end
```

This was all I added. It just checks to see if it's the first of the month, and if it is, it makes 3 new houses.